ads/README.md

```diff
@@ -130,6 +130,22 @@ Here are some factors that affect whether the resize will be executed:
 - Whether the resize is requested for a currently active ad;
 - Whether the resize is requested for an ad below the viewport or above the viewport.
 
+
+### Support for variable size ad requests
+In order to support multi-size ad requests, AMP accepts an optional `data` param to `window.context.renderStart` which will automatically invoke request resize with the width and height passed.
+In case the resize is not successful, AMP will horizontally and vertically center align the creative within the space initially reserved for the creative.
+This utility makes it convenient for ad networks to support mult-size ad requests in AMP.
+#### Example
+```javascript
+window.context.renderStart(data);
```

Comment:
@yieldmo-rao what's a valid reason for an ad to do that? Why wouldn't the ad respond (on renderStart) with 300x500 in the first place?